src/qml/QFieldCloudScreen.qml

```diff
@@ -350,13 +350,18 @@ Page {
                                   case QFieldCloudProjectsModel.ProjectStatus.Idle:
                                     break
                                   case QFieldCloudProjectsModel.ProjectStatus.Downloading:
-                                    switch (PackagingStatus) {
-                                      case QFieldCloudProjectsModel.PackagingFinishedStatus:
-                                        status = qsTr( 'Downloading, %1% fetched…' ).arg( Math.round(DownloadProgress * 100) )
-                                        break;
-                                      default:
-                                        status = qsTr('QFieldCloud is preparing the latest data just for you. This might take some time, please hold tight…')
-                                        break;
+                                    if (PackagingStatus === QFieldCloudProjectsModel.PackagingBusyStatus) {
```

Comment:
Ideally we can avoid repeating this as it is already in `QFieldCloudPopup.qml` bit I cannot suggest anything that is easy and worth it.